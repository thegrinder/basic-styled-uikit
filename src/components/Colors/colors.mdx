---
name: Colors
menu: Components
route: /components/colors
---

## **ColorsProvider**

