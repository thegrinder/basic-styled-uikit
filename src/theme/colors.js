import { getColors } from './themeSelectors';

export const getColor = color => ({ theme }) => getColors(theme)[color];

const colors = {
  neutral0: '#fff',
  neutral100: '#f8f9fa',
  neutral400: '#ced4da',
  neutral600: '#868e96',
  neutral1000: '#161616',

  primary400: '#42A5F5',
  primary600: '#1E88E5',
  success400: '#66BB6A',
  success600: '#43A047',
  warning400: '#FFEE58',
  warning600: '#FDD835',
  danger400: '#FF8A65',
  danger600: '#F4511E',
};

export default colors;
