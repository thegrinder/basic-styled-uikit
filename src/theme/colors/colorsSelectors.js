import { getColors } from '../themeSelectors';

// eslint-disable-next-line
export const getColor = color => ({ theme }) => getColors(theme)[color];
