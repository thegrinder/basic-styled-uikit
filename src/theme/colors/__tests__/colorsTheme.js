import colors from '../colorsTheme';

describe('colorsTheme', () => {
  it('should be defined', () => {
    expect(colors).toBeDefined();
    expect(colors).toMatchSnapshot();
  });
});
