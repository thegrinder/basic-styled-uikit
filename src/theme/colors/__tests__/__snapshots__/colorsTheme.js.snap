// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colorsTheme should be defined 1`] = `
Object {
  "black": "#161616",
  "danger100": "#ffcdd2",
  "danger200": "#ef9a9a",
  "danger300": "#e57373",
  "danger400": "#ef5350",
  "danger50": "#ffebee",
  "danger500": "#f44336",
  "danger600": "#e53935",
  "danger700": "#d32f2f",
  "danger800": "#c62828",
  "danger900": "#b71c1c",
  "gray100": "#f8f9fa",
  "gray200": "#e9ecef",
  "gray300": "#dee2e6",
  "gray400": "#ced4da",
  "gray450": "#bec5cc",
  "gray50": "#fafafa",
  "gray500": "#adb5bd",
  "gray600": "#868e96",
  "gray700": "#495057",
  "gray800": "#343a40",
  "gray900": "#212529",
  "inherit": "inherit",
  "primary100": "#bbdefb",
  "primary200": "#90caf9",
  "primary300": "#64b5f6",
  "primary400": "#42a5f5",
  "primary50": "#e3f2fd",
  "primary500": "#2196f3",
  "primary600": "#1e88e5",
  "primary700": "#1976d2",
  "primary800": "#1565c0",
  "primary900": "#0d47a1",
  "success100": "#c8e6c9",
  "success200": "#a5d6a7",
  "success300": "#81c784",
  "success400": "#66bb6a",
  "success50": "#e8f5e9",
  "success500": "#4caf50",
  "success600": "#43a047",
  "success700": "#388e3c",
  "success800": "#2e7d32",
  "success900": "#1b5e20",
  "transparent": "transparent",
  "warning100": "#fff9c4",
  "warning200": "#fff59d",
  "warning300": "#fff176",
  "warning400": "#ffee58",
  "warning50": "#fffde7",
  "warning500": "#ffeb3b",
  "warning600": "#fdd835",
  "warning700": "#fbc02d",
  "warning800": "#f9a825",
  "warning900": "#f57f17",
  "white": "#fff",
}
`;
